http/headers/feature-policy.json

```diff
@@ -212,7 +212,7 @@
                 "version_added": false
               },
               "samsunginternet_android": {
-                "version_added": false
+                "version_added": "10.0"
```

Comment:
Behind a pref in Chrome, enabled in Samsung?

Reply:
Hmm, I was discarding all entries with flags since we generally don't support them. I think there is a bug in my script here for those.